subjects/DPxAI/first-hello/README.md

```diff
@@ -0,0 +1,93 @@
+## First hello
+
+> JSPowered Mode
+
+### Context
+
+Your robot winked, but you can make it talk too! Or at least, make it say his first hello to the world!
+
+> Follow the instructions, ask your peers if you are stuck and stay motivated because you are close to your goal!
+> Follow every hint you have in the subject!
+> Continue on the code from last exercise, and change the file names!
+
+### Instructions
+
+Now that you know how to make your creation move, what about making it communicate its first words to the world?
+
+After finishing these tasks, you will be able to display and hide `Hello World` in the `torso` of your robot by clicking on a second button.
+
+#### Task 1:
+
+Let's put a second button in the top right corner of the page, that will add some text when clicked. Add it in the HTML structure:
+
+```html
+<button id="speak-button">Click me to speak</button>
+```
+
+Add the button style in the CSS file:
+
+```css
```

Comment:
did they already get an idea about css what is it and how it work ? because providing css like that without given them any idea about it will make them feel that they always messing something.

Reply:
Yes, and they are permitted to use GA to learn more about it